### DISCUSSION TITLE
Editable Columns Plugin -> Using Editable Select with relationships

### DISCUSSION THREAD

**Discussion by ceeckhout** [Laravel-Backpack/community-forum#358] [Q&A (Help)] [answered]
Related to the next post; https://stackoverflow.com/questions/74825271/editable-columns-plugin-using-editable-select-gives-error , I get the same error. 

...
use \Backpack\EditableColumns\Http\Controllers\Operations\MinorUpdateOperation;
...
    protected function setupListOperation()
    {
        ...
        // editable_select
        CRUD::column('supplier1')
            ->type('editable_select')
            ->options(\App\Models\Supplier::all()->pluck('name', 'id')->toArray());
        ...
    }

So I wonder if it really works in case of a relationship with another model. Has anyone already succeeded? If so, any idea what I could be doing wrong?

Thanks in advance!

**Comment by pxpm** [comment, marked as answer]
Hello @ceeckhout 

Editable columns doesn't understand relationships like backpack fields do. 

You should provide the `database column name` of the relationship, in your case, probably `supplier1_id` so it should be:

```php
CRUD::column('supplier1_id') // use the database column name
->type('editable_select')
->options(\App\Models\Supplier::all()->pluck('name', 'id')->toArray());
```

Cheers

**Reply by ceeckhout** [reply]
Thanks a lot!!
